apps/wallet/src/features/Send/InputTransaction/internal.ts

```diff
@@ -1,84 +1,3 @@
 import { Networks } from '@walless/core';
-import { checkValidAddress } from 'utils/transaction';
-
-import { txContext } from '../context';
-
-export const totalCheckFieldsToContinue = () => {
```

Comment:
you removed the total check and disabled the button, make sure that's the correct UX as we defined or expected